frontend/.eslintrc

```diff
@@ -119,9 +120,7 @@
     },
     // Check for import violation in all JS-like files
     "import/resolver": {
-      "node": {
-        "extensions": [".js", ".jsx", ".ts", ".tsx", ".json"]
-      }
```

Comment:
Is this just obsolete?

Reply:
NGL @kmcgrady figured out this config so I'm not sure if my understanding is right/complete, but I thought this is obsolete because using `eslint-import-resolver-typescript` via `"typescript": true` config covers these extensions by default ([doc](https://github.com/import-js/eslint-import-resolver-typescript#extensions))